samples/SkiaJvmSample/build.gradle.kts

```diff
@@ -56,6 +56,7 @@ val casualRun = tasks.named<JavaExec>("run") {
     systemProperty("skiko.win.exception.logger.enabled", "true")
     systemProperty("skiko.win.exception.handler.enabled", "true")
     jvmArgs?.add("-ea")
+    // jvmArgs?.add("-Xcheck:jni")
```

Comment:
Leftover.

Reply:
left it there intentionally, as reminder to check jni from time to time